pkg/apis/vpc/v1alpha1/networkinfo_types.go

```diff
@@ -29,6 +29,13 @@ type NetworkInfoList struct {
 	Items           []NetworkInfo `json:"items"`
 }
 
+type NetworkStackType string
+
+const (
+	FULL_STACK_VPC  NetworkStackType = "FULL_STACK_VPC"
```

Comment:
We should use Golang style for constants and K8s style for the values, and so it should be "fullStackVPC", and "FullStackVPC". @tnqn @dantingl